examples/app-python/example_app.py

```diff
@@ -208,6 +246,8 @@ def new():
     # Note : SatOS-Payload-SDK supports sequence upto 16 characters long
     app.mount_sequence("HelloWorld", ctl.handle_hello_world)
     app.mount_sequence("HelloFriend", ctl.handle_hello_friend)
+    app.mount_sequence("ThreadStart", ctl.handle_demo_thread_start)
```

Comment:
Can we have some meaningful names for the sequences, like StartImaging, StopImaging etc ? Also we should get rid of examples of meaningless sequences like "HelloWorld", "HelloFriend" etc. Doesn't look enterprise ready example code.

Also, let's not do printing numbers kind of trivial work in handler functions. It's better to print a log msg and come out. We can put a detail comment there for users to stuff some meaningful code

Reply:
This will be handled in separate ticket here: https://antaris.atlassian.net/browse/AT-5012